lib/sequent/migrations/versions.rb

```diff
@@ -10,7 +10,9 @@ class Versions < Sequent::ApplicationRecord
 
       def self.migration_sql
         <<~SQL.chomp
-          CREATE TABLE IF NOT EXISTS #{table_name} (version integer NOT NULL, CONSTRAINT version_pk PRIMARY KEY(version));
+          CREATE TABLE IF NOT EXISTS #{table_name} (version integer NOT NULL, target_projectors text[] DEFAULT '{}'::text[], target_records text[] DEFAULT '{}'::text[], CONSTRAINT version_pk PRIMARY KEY(version));
+          ALTER TABLE #{table_name} ADD COLUMN IF NOT EXISTS target_projectors text[] DEFAULT '{}'::text[];
+          ALTER TABLE #{table_name} ADD COLUMN IF NOT EXISTS target_records text[] DEFAULT '{}'::text[];
```

Comment:
It is now possible to use regular active record migrations in the `db/migrate` directory. The SQL here should actually be removed.

Reply:
As discussed. We'll leave this as is for now since the table for `Sequent::Migrations::Versions` lives on the view schema.